openedx/core/djangoapps/content/course_overviews/api.py

```diff
@@ -2,12 +2,18 @@
 CourseOverview internal api
 """
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
-
 from openedx.core.djangoapps.content.course_overviews.serializers import (
     CourseOverviewBaseSerializer,
 )
 
 
+def get_course_overview(course_id):
```

Comment:
I added this to try and reduce the use of the CourseOverview model directly in the certificates app. There _is_ another function in this API file that retrieves course_overviews from a list of `course_ids`. However, the return value is a serialized list of dicts (of the CourseOverviews).

It's much easier to work with the object itself... but I can remove this and adapt to working with the list/dict if there are strong feelings of opposition.